helm/observability-operator/files/alertmanager/url-template.tmpl.helm-template

```diff
@@ -0,0 +1,18 @@
+{{`
+{{ define "__alerturl" }}
+`}}{{ .alertmanager.grafanaAddress }}{{`/alerting/Mimir/{{ .CommonLabels.alertname }}/find
+{{ end }}
+
+{{ define "__dashboardurl" -}}
+{{ if match "^https://.+" (index .Alerts 0).Annotations.dashboard }}{{ (index .Alerts 0).Annotations.dashboard }}
+{{ else }}
+`}}{{ .alertmanager.grafanaAddress }}{{`/d/{{ (index .Alerts 0).Annotations.dashboard }}
+{{ end }}
+{{- end }}
+
+{{ define "__queryurl" }}
+`}}{{ .alertmanager.grafanaAddress }}{{`/alerting/Mimir/{{ .CommonLabels.alertname }}/find
+{{ end }}
+
+{{ define "__runbookurl" -}}https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/{{ (index .Alerts 0).Annotations.opsrecipe }}{{- end }}
```

Comment:
I think this could be moved to the notification template there is no need for some escaping on this one

Reply:
Just tried to make things consistent by having all the urls here, but I can move this

Reply:
I understand but let's try to get rid of this file :D